### PR TITLE
TC_02.02.008 | Header >Verify Jenkins Logo and Title on welcome page is visible and clickable

### PR DESCRIPTION
**Pull request by IliaKhudiakov**
https://trello.com/c/PPfgFg5t/243-tc0202008-header-verify-jenkins-logo-and-title-on-welcome-page-is-visible-and-clickable

### CODE REVIEW

File: cypress/e2e/VerifyJenkinsLogoInWelcomePage.cy.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/// <reference types = "cypress"/>
+
+const LOCAL_PORT = Cypress.env('local.port');
+const LOCAL_HOST = Cypress.env('local.host');
+
+describe ('Verify Jenkins Logo', () => {
+
+    it ('Verify Jenkins Logo and Title on welcome page is visible and clickable', function() {
```
Comment (S-Lanchik) [inline cypress/e2e/VerifyJenkinsLogoInWelcomePage.cy.js:8]: Add number to your test name

File: cypress/e2e/VerifyJenkinsLogoInWelcomePage.cy.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        cy.get('#jenkins-name-icon').should('be.visible')
+        cy.get('#jenkins-name-icon').should('have.attr', 'src')
+        cy.get('#jenkins-name-icon').click
+        cy.url().should('eq', `http://${LOCAL_HOST}:${LOCAL_PORT}/`)
+        cy.get('h1').should('have.text', 'Welcome to Jenkins!')
```
Comment (S-Lanchik) [inline cypress/e2e/VerifyJenkinsLogoInWelcomePage.cy.js:13]: Put 'Welcome to Jenkins!' to fixtures

Comment (IliaKhudiakov) [reply cypress/e2e/VerifyJenkinsLogoInWelcomePage.cy.js]: I have corrected. Thanks!

Comment (S-Lanchik) [reply cypress/e2e/VerifyJenkinsLogoInWelcomePage.cy.js]: Good! Just change the file name to start with small letter (verifyJenkinsLogoInWelcomePage.cy.js)

**Review by S-Lanchik — APPROVED** [review]

**Review by Devgeni-SQA — APPROVED** [review]

**Pull request merged**